BraveRewardsUI/Settings/Grants/SettingsGrantSectionView.swift

```diff
@@ -50,10 +50,11 @@ class SettingsGrantSectionView: SettingsSectionView {
     switch type {
     case .ads(let amount):
       iconImageView.image = UIImage(frameworkResourceNamed: "icn-ads")
+      
       if let amount = amount {
-        textLabel.text = String(format: Strings.SettingsAdsGrantText, "\(amount) \(Strings.BAT) ")
+        textLabel.text = "\(amount) \(Strings.BAT) " + Strings.SettingsAdsGrantText
```

Comment:
Can we just add this to the string formatter like so:
`"\(amount) \(Strings.BAT) \(Strings.SettingsAdsGrantText)"`